librz/signature/meson.build

```diff
@@ -52,7 +52,10 @@ pkgconfig_mod.generate(rz_sign,
     'rz_type',
     'rz_flag',
   ],
-  description: 'rizin foundation libraries'
+  description: 'rizin foundation libraries',
```

Comment:
I would add this to all .pc files, not only this one. Also, the .cmake files as well. It shouldn't be too big of a change in cmake, hopefully.